docs/infrastructure/deployment-targets/dynamic-infrastructure/new-octopustarget.md

```diff
@@ -6,6 +6,12 @@ position: 50
 
 Targets defined by step packages can be created either by PowerShell or bash functions available in any Octopus script-running context. Not all targets are defined by step packages. The complete list of targets defined by step packages is available below.
 
+:::warning
+**Deprecated**
+
+Creating deployment targets using the `New-OctopusTarget` function has been deprecated in favour of using [Cloud Target Discovery](/docs/infrastructure/deployment-targets/cloud-target-discovery/index.md).
```

Comment:
One of the drivers of wanting to add this callout was an issue we found where the shape of the ECS cluster inputs changed between versions of the ECS cluster target. We may want something additional here in terms of stronger specific language indicating the inputs may not be up to date because the feature is deprecated.

Reply:
I'd recommend moving the warning to Line 6. This will make the call-out the first thing customers see.

Reply:
When moving the call-out, I'd also consider whether the additional call-out is relevant:

![image](https://user-images.githubusercontent.com/1418993/170964009-376ab418-8899-4635-8a5c-270a257dd62e.png)

It might be better to simplify the first section, something like this (feel free to change the wording):

```text
:::warning
**Deprecated**

Creating deployment targets using the `New-OctopusTarget` function has been deprecated in favor of using [Cloud Target Discovery](/docs/infrastructure/deployment-targets/cloud-target-discovery/index.md).
:::

In **Octopus 2021.3**, a new architecture for deployments and steps targets was developed, known as **step packages**. Targets defined by step packages can be created either by PowerShell or bash functions available in any Octopus script-running context. Not all targets are defined by step packages. The complete list of targets defined by step packages is available below.

To create a target defined by a step package, you will need to know the `target identifier`, and the `inputs` required by the target. These can currently be found in the following locations:
```